src/qml/components/BlockClock.qml

```diff
@@ -95,13 +99,24 @@ Item {
 
     PeersIndicator {
         anchors.top: subText.bottom
-        anchors.topMargin: 20
+        anchors.topMargin: dial.width / 10
         anchors.horizontalCenter: root.horizontalCenter
         numOutboundPeers: nodeModel.numOutboundPeers
         maxNumOutboundPeers: nodeModel.maxNumOutboundPeers
+        indicatorDimensions: dial.width * (3/200)
+        indicatorSpacing: dial.width / 40
         paused: root.paused
     }
 
+    NetworkIndicator {
+        id: networkIndicator
+        property int topMargin: dial.width * (3/20)
+        anchors.top: dial.bottom
+        textSize: dial.width * (3/40)
```

Comment:
I understand why you did it here to match the original design, but I think making the size of this component depend on the size of the dial creates the wierd logic with having to know more about the parents layout than it should.